RevenueCatUI/Helpers/Package+VariableDataProvider.swift

```diff
@@ -94,41 +115,44 @@ extension Package: VariableDataProvider {
         return self.introDuration(locale)
     }
 
-    func localizedPricePerPeriod(_ locale: Locale) -> String {
+    func localizedPricePerPeriod(_ locale: Locale, showZeroDecimalPlacePrices: Bool = false) -> String {
         guard let period = self.storeProduct.subscriptionPeriod else {
-            return self.localizedPrice
+            return self.localizedPrice(showZeroDecimalPlacePrices: showZeroDecimalPlacePrices)
         }
 
         let unit = Localization.abbreviatedUnitLocalizedString(for: period, locale: locale)
-        return "\(self.localizedPrice)/\(unit)"
+        return "\(self.localizedPrice(showZeroDecimalPlacePrices: showZeroDecimalPlacePrices))/\(unit)"
     }
 
-    func localizedPricePerPeriodFull(_ locale: Locale) -> String {
+    func localizedPricePerPeriodFull(_ locale: Locale, showZeroDecimalPlacePrices: Bool = false) -> String {
         guard let period = self.storeProduct.subscriptionPeriod else {
-            return self.localizedPrice
+            return self.localizedPrice(showZeroDecimalPlacePrices: showZeroDecimalPlacePrices)
         }
 
         let unit = Localization.unitLocalizedString(for: period, locale: locale)
-        return "\(self.localizedPrice)/\(unit)"
+        return "\(self.localizedPrice(showZeroDecimalPlacePrices: showZeroDecimalPlacePrices))/\(unit)"
     }
 
-    func localizedPriceAndPerMonth(_ locale: Locale) -> String {
+    func localizedPriceAndPerMonth(_ locale: Locale, showZeroDecimalPlacePrices: Bool = false) -> String {
         if !self.isSubscription || self.isMonthly {
-            return self.localizedPricePerPeriod(locale)
+            return self.localizedPricePerPeriod(locale, showZeroDecimalPlacePrices: showZeroDecimalPlacePrices)
         } else {
             let unit = Localization.abbreviatedUnitLocalizedString(for: .init(value: 1, unit: .month),
                                                                    locale: locale)
-            return "\(self.localizedPricePerPeriod(locale)) (\(self.localizedPricePerMonth)/\(unit))"
+            // swiftlint:disable:next line_length
+            return "\(self.localizedPricePerPeriod(locale, showZeroDecimalPlacePrices: showZeroDecimalPlacePrices)) (\(self.localizedPricePerMonth(showZeroDecimalPlacePrices: showZeroDecimalPlacePrices))/\(unit))"
```

Comment:
See below for how to write this without disabling swiftlint 👇

Reply:
Done!